assets/src/block-validation/amp-validation-status/status-notification.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, ExternalLink } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import AMPValidationErrorsKeptIcon from '../../../images/amp-validation-errors-kept.svg';
+import { BLOCK_VALIDATION_STORE_KEY } from '../store';
+import { StatusIcon } from '../icon';
+import { SidebarNotification } from '../../block-editor/components/sidebar-notification';
+
+/**
+ * AMP validation status notification component.
+ */
+export default function AMPValidationStatusNotification() {
+	const { autosave, savePost } = useDispatch( 'core/editor' );
+
+	const {
+		ampCompatibilityBroken,
+		fetchingErrorsRequestErrorMessage,
+		hasValidationErrors,
+		isDraft,
+		isEditedPostNew,
+		isFetchingErrors,
+		reviewLink,
+	} = useSelect( ( select ) => ( {
+		ampCompatibilityBroken: select( BLOCK_VALIDATION_STORE_KEY ).getAMPCompatibilityBroken(),
+		fetchingErrorsRequestErrorMessage: select( BLOCK_VALIDATION_STORE_KEY ).getFetchingErrorsRequestErrorMessage(),
+		hasValidationErrors: select( BLOCK_VALIDATION_STORE_KEY ).getValidationErrors()?.length > 0,
+		isDraft: [ 'draft', 'auto-draft' ].indexOf( select( 'core/editor' )?.getEditedPostAttribute( 'status' ) ) !== -1,
+		isEditedPostNew: select( 'core/editor' ).isEditedPostNew(),
+		isFetchingErrors: select( BLOCK_VALIDATION_STORE_KEY ).getIsFetchingErrors(),
+		reviewLink: select( BLOCK_VALIDATION_STORE_KEY ).getReviewLink(),
+	} ), [] );
+
+	if ( isFetchingErrors ) {
+		return null;
+	}
+
+	if ( fetchingErrorsRequestErrorMessage ) {
+		return (
+			<SidebarNotification
+				icon={ <AMPValidationErrorsKeptIcon /> }
+				message={ fetchingErrorsRequestErrorMessage }
+				action={
+					<Button
+						isLink
+						onClick={ isDraft
+							? () => savePost( { isPreview: true } )
+							: () => autosave( { isPreview: true } )
+						}
+					>
+						{ __( 'Try again', 'amp' ) }
+					</Button>
+				}
+			/>
+		);
+	}
+
+	if ( ampCompatibilityBroken ) {
+		return (
+			<SidebarNotification
+				icon={ <AMPValidationErrorsKeptIcon /> }
+				message={ __( 'AMP blocked from validation issues marked kept.', 'amp' ) }
```

Comment:
The wording may not be clear to the user here. How about something like:

```suggestion
				message={ __( 'AMP is disabled due to validation issues being marked as kept.', 'amp' ) }
```

cc @westonruter

Reply:
How about rather:

```suggestion
				message={ __( 'AMP is disabled due to invalid markup being kept.', 'amp' ) }
```

Reply:
The issue count should probably be indicated as well.

Reply:
See 8b42ad96e385431f71f561818126ef8d0fcd8cd9